Procfile

```diff
@@ -5,9 +5,9 @@ worker_buildapi_pending: newrelic-admin run-program celery worker -A treeherder
 worker_buildapi_running: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q buildapi_running --concurrency=5
 worker_buildapi_4hr: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q buildapi_4hr --concurrency=1
 worker_store_pulse_jobs: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_jobs --concurrency=3
-worker_store_pulse_pushes: newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_pushes --concurrency=3
```

Comment:
Let's not change anything in this file yet.  We'll do it in a follow-up PR.  And we'll need to change back anything that references this.